tests/core/daemon/test_daemon.py

```diff
@@ -51,6 +39,10 @@ async def get_temp_keyring(self):
         with TempKeyring() as keychain:
             yield keychain
 
+    @pytest.fixture(scope="function")
+    async def get_b_tools_1(self, get_temp_keyring):
+        return await create_block_tools_async(constants=test_constants_modified, keychain=get_temp_keyring)
```

Comment:
This uses the same keyring as `get_b_tools` while the existing global code had two keyrings.  Is this ok?  (@paninaro)

But...  I do appreciate the elimination of the global bits.

Reply:
Should be ok I think. I believe this is similar to the approach I was taking in one of my stale PRs to re-introduce/fix the disabled legacy keyring test. In any event it brings us closer to eliminating the globals.